src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -149,7 +151,9 @@ function MdAutocomplete () {
       floatingLabel:    '@?mdFloatingLabel',
       autoselect:       '=?mdAutoselect',
       menuClass:        '@?mdMenuClass',
-      inputId:          '@?mdInputId'
+      inputId:          '@?mdInputId',
+      noBlurOnEscape:   '=?mdNoBlurOnEscape',
```

Comment:
Surely it's possible, but I couldn't find an example of another boolean attribute that defaults to true. 

I could rewrite it as mdEscapeOptions which defaults to "blur clear" which could be clearer to the user, but then it could be confusing in terms of what md-escape-options="" would produce. If it's blank, would it default to "blur clear" or would it not do anything?

I think the current syntax isn't too confusing from end-user perspective, but I can understand how it could look confusing from coding perspective.